tests/compute/test_heterograph.py

```diff
@@ -2092,6 +2153,6 @@ def test_reverse(index_dtype):
     # test_isolated_ntype()
     # test_bipartite()
     # test_dtype_cast()
-    # test_reverse("int32")
+    test_reverse("int32")
```

Comment:
Copy @BarclayII and @VoVAllen for visibility. I think we don't need to pass the data type?

Reply:
For CI, it will automatically test int32 and int64. For offline testing, we need to specify the type.